tests/Hangfire.Tags.PostgreSql.Tests/Hangfire.Tags.PostgreSql.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This is causing the CI to fail. Can we use 2.2 instead?

Reply:
yea, rolling it back